geode-cq/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/DurableClientCommandsDUnitTest.java

```diff
@@ -112,9 +124,40 @@ public void testListDurableClientCqs() {
     csb = new CommandStringBuilder(CliStrings.LIST_DURABLE_CQS);
     csb.addOption(CliStrings.LIST_DURABLE_CQS__DURABLECLIENTID, CLIENT_NAME);
     commandString = csb.toString();
-    String errorMessage =
-        CliStrings.format(CliStrings.LIST_DURABLE_CQS__NO__CQS__FOR__CLIENT, CLIENT_NAME);
-    gfsh.executeAndAssertThat(commandString).statusIsError().containsOutput(errorMessage);
+
+    gfsh.executeAndAssertThat(commandString).statusIsError()
+        .tableHasColumnWithExactValuesInAnyOrder("CQ Name",
+            CliStrings.format(CliStrings.LIST_DURABLE_CQS__NO__CQS__FOR__CLIENT, CLIENT_NAME),
+            CliStrings.format(CliStrings.LIST_DURABLE_CQS__NO__CQS__REGISTERED));
+  }
+
+  @Test
+  public void testListDurableClientCqsWhenNoneExist() {
+    CommandStringBuilder csb = new CommandStringBuilder(CliStrings.LIST_DURABLE_CQS);
+    csb.addOption(CliStrings.LIST_DURABLE_CQS__DURABLECLIENTID, CLIENT_NAME);
+    String commandString = csb.toString();
+
+    gfsh.executeAndAssertThat(commandString).statusIsError()
+        .tableHasColumnWithExactValuesInAnyOrder("CQ Name",
```

Comment:
I think we should deprecate these methods in CommandResultAssert. these are used for the old CommandResult, for the ResultModel, the other assertions are much easier to understand.

Reply:
Fixed